addons/api/mirage/config.js

```diff
@@ -632,7 +632,23 @@ function routes() {
       { queryParams: { credential_store_id: credentialStoreId } },
     ) => credentialLibraries.where({ credentialStoreId }),
   );
-  this.get('/credential-libraries/:id');
+  this.get(
+    '/credential-libraries/:id',
+    function ({ credentialLibraries }, { params: { id } }) {
+      const library = credentialLibraries.find(id);
+      // BE API only updates these fields if there's an update mask, otherwise they remain unmodified
```

Comment:
Sorry I just saw this PR but shouldn't this be handled in the mirage serializers?

Reply:
I tried to mimic the API. We send these in the PATCH request, but we don't get these back from the GET req. If I were to add these in the serializer, wouldn't that change the flow of how this is done in the API?

Reply:
Are you referring to our our application serializers or mirage serializers? The mirage serializers would just handle how the data shows up in mirage so it shouldn't affect anything to do with the API?

Reply:
I was referring to the Mirage serializer and trying to align it with how it works when a real API is used. I don't know if we usually add these in Mirage serializers. I can move it there, but my reasoning was that we don't serialize this when a real API is involved. So, I did this in the GET request.

Reply:
I'm not sure I understand, don't we normally do these type of of things normally in the mirage serializers? To get it to match how the real API looks like?

The mirage configs I thought were mainly for changing what the actual data is, not how it should look like

Reply:
Had offline discussion, and looks like that's where this processing should exist - addressed it here